pkgs/development/python-modules/ratelim/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, decorator
+}:
+
+buildPythonPackage rec {
+  pname = "ratelim";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "themiurgo";
+    repo = pname;
+    rev = "master";
+    sha256 = "0qznfp5jayp612yv7zrxz1mfp34n1zqa6aj8i1zjrrhiv3zfq7fx";
+  };
+
+  propagatedBuildInputs = [
+    decorator
+  ];
+
+  pythonImportsCheck = [ "ratelim" ];
```

Disable the checkPhase if the library doesn't have any tests.

```suggestion
  pythonImportsCheck = [ "ratelim" ];
  
  # package has no tests
  doCheck = false;
```

But we could run https://github.com/themiurgo/ratelim/blob/0.1.1/ratelim/__init__.py#L71-L90 instead?